virtualfish/virtual.fish

```diff
@@ -47,10 +47,27 @@ function __vf_activate --description "Activate a virtualenv"
         echo "You need to specify a virtualenv."
         return 1
     end
-    if not [ -d $VIRTUALFISH_HOME/$argv[1] ]
-        echo "The virtualenv $argv[1] does not exist."
-        echo "You can create it with `vf new $argv[1]`."
-        return 2
+
+    if echo $argv[1] | grep -q /
```

Comment:
I think this can be done within Fish and without piping to `grep`:

    if string match "*/*" $argv[1]

Unless I'm reading the following output wrong, I think doing this operation using Fish's `string match` is nearly three orders of magnitude faster:

```
~ ➤ time string match "*/*" "~/.virtualenvs"

________________________________________________________
Executed in    5.00 micros    fish           external
   usr time    4.00 micros    4.00 micros    0.00 micros
   sys time    2.00 micros    2.00 micros    0.00 micros

~ ➤ time echo "~/.virtualenvs" | grep -q /

________________________________________________________
Executed in    3.75 millis    fish           external
   usr time    0.89 millis  184.00 micros    0.71 millis
   sys time    2.46 millis  766.00 micros    1.70 millis
```